client/src/components/LogHealthSlider.jsx

```diff
@@ -83,7 +83,19 @@ const useStyles = makeStyles(theme => ({
 const LogHealthSlider = () => {
   const classes = useStyles();
   const history = useHistory();
-  const oneThird = window.innerWidth / 3; // Note: doesn't support window resizing
+  const [oneThird, setOneThird] = useState(0);
+
+  useEffect(() => {
+    // grab slider element and use its width to calculate the oneThird Value
+    let slider = document.querySelector('.makeStyles-item-250');
```

Comment:
instead of using an auto-generated class here, we should go in and give whichever item you're selecting a class ourselves